scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -298,7 +298,7 @@ def setup_layout(self):
         add_runs_page_icon = icons.get_icon("mdi.plus-circle-outline")
```

Comment:
```suggestion
        add_runs_page_icon = icons.get_icon("mdi.sigma")
```

Reply:
I've removed the circle, but kept plus as sigma is used elsewhere in Mantid to denote processing